.github/workflows/CI.yml

```diff
@@ -71,13 +71,14 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1'
+          version: '1.6'
       - run: |
           julia --project=docs -e '
             using Pkg
             Pkg.develop(PackageSpec(path=pwd()))
             Pkg.instantiate()
             include("docs/make.jl")'
         env:
+          JULIA_PKG_SERVER: ""
```

Comment:
I know nothing about CI, what does this do?

Reply:
See [this thread](https://julialang.slack.com/archives/C67TK21LJ/p1629306333256900) for proper answer to this question. 

TLDR: The package server can be out of date, meaning that the CI (depending on how its built) can miss recent releases, introducing compat issues that don't actually exist